core/commands/playerround/create/create_playerround_command_handler.py

```diff
@@ -18,23 +17,13 @@ def __init__(self):
         self.round_repository = RoundRepository(Round)
 
     def handle(self, command: CreatePlayerroundCommand) -> Result[PlayerroundDto]:
-        serializer = CreatePlayerroundCommandSerializer(
-            data={
-                'playerid': command.playerid,
-                'roundid': command.roundid,
-                'totalscore': '',
-            }
-        )
-        if not serializer.is_valid():
-            return Result.fail(serializer.errors, status_code=400)
-
         if not self.round_repository.round_exists(roundid=command.roundid):
```

Comment:
This round_exists should be obsolete when we define the roundid REFERENCE as a PrimaryRelatedKeyField() -- just like the golfcourse PR open atm

Reply:
I just looked at "a PrimaryRelatedKeyField() -- just like the golfcourse PR open atm". That solution places DB code in the serializer instead of the repository, and i think that obscures responsibility as the DTO should just be a simple mapping of the data we return, and not perform more logic

Reply:
We can abstain from using it. I realize we are putting dB access code outside of repositories.

On the plus side tho:
* we are enforcing reference existence at db level, and not based on our written validation
* it's also more lean
(1 line of code, compared to one method for each object--location_exists,player_exists,...)

Reply:
To avoid repeating the object_exists(**kwargs) on each model repository, we could move it to BaseRepository and have:

@abstractmethod
def exists(**kwargs) -> bool:
        """Give any key,value pair identifier to look in the database."""
        pass
        
This keeps DB access code in repositories and limits repetitive method across all repos. I think this would be a good solution

Reply:
suggestion for another PR

Reply:
Good solution, let's do that

Reply:
I have created task on milanote